das/daser.go

```diff
@@ -22,13 +22,16 @@ var log = logging.Logger("das")
 const (
 	// samplingCoordinator will paginate headers into jobs with fixed size.
 	//  samplingRange is the maximum amount of headers processed in one job.
-	samplingRange = 64
+	samplingRange = 100
```

Comment:
My point for `range` is because that items are consecutive in between `from` and `to`. I feel like size would be nicer if it was a limit for amount of items in batch, but not necessarily having an order. 
Also header package has method `GetRangeByHeight` that will be introduced into DAS'er and will use this constant